doc/python/BACCAB.py

```diff
@@ -1,24 +1,24 @@
-from printer import Format,xpdf
-from ga import Ga
-from mv import Com
+from __future__ import absolute_import, division
+from __future__ import print_function
+from galgebra.printer import Format, xpdf
+from galgebra.ga import Ga
```

Comment:
If the users install galgebra from pip, the way of importing the modules has to conform to the standard "namespaced" way instead of just importing modules from a global namespace. This might break some old code but it's a necessary process.

Reply:
agreed